parser/src/schema/body.rs

```diff
@@ -60,6 +69,18 @@ impl From<Body> for SchemaBody {
                 content,
                 mime: mime::TEXT_XML,
             },
+            Body::Gzip(path) => SchemaBody::Binary { 
+                path, 
+                mime: Mime::from_str("application/gzip").unwrap()
```

Comment:
```suggestion
                mime: Mime::from_str("application/gzip").unwrap(),
```

`cargo fmt` suggestion.